Content.Server/StationEvents/Events/RandomSentienceRule.cs

```diff
@@ -45,24 +63,14 @@ protected override void Started(EntityUid uid, RandomSentienceRuleComponent comp
         var kind2 = groupList.Count > 1 ? groupList[1] : "???";
         var kind3 = groupList.Count > 2 ? groupList[2] : "???";
 
-        foreach (var target in targetList)
-        {
-            var station = StationSystem.GetOwningStation(target);
-            if(station == null)
-                continue;
-            stationsToNotify.Add((EntityUid) station);
-        }
-        foreach (var station in stationsToNotify)
-        {
-            ChatSystem.DispatchStationAnnouncement(
-                station,
-                Loc.GetString("station-event-random-sentience-announcement",
-                    ("kind1", kind1), ("kind2", kind2), ("kind3", kind3), ("amount", groupList.Count),
-                    ("data", Loc.GetString($"random-sentience-event-data-{RobustRandom.Next(1, 6)}")),
-                    ("strength", Loc.GetString($"random-sentience-event-strength-{RobustRandom.Next(1, 8)}"))),
-                playDefaultSound: false,
-                colorOverride: Color.Gold
-            );
-        }
+        ChatSystem.DispatchStationAnnouncement(
+            (EntityUid) station, // cast from nullable. If this was null, we wouldn't be here
```

Comment:
I'm surprised TryGetRandomStation doesn't handle this with NotNullWhen, but you should be able to do:

```suggestion
            station!,
```